charts/kyverno/templates/networkpolicy.yaml

```diff
@@ -14,7 +14,17 @@ spec:
   - Ingress
   ingress:
   - from:
-    ports:
-    - protocol: TCP
-      port: 9443 # webhook access
+      namespaceSelector:
+        matchExpressions:
+          {{- toYaml .Values.networkPolicy.namespaceExpressions | nindent 8 }}
+        matchLabels:
```

Comment:
I'm not sure we should force a presence of a label `app: kyverno` to match the selector (same thing for the `podSelector`). I think the user should be able to choose which all labels to match.